### DISCUSSION TITLE
It is time to remove Tiddlyfox documentation

### DISCUSSION THREAD

**Discussion by ibnishak** [TiddlyWiki/TiddlyWiki5#6323] [General]
Tiddlyfox is no more available in the firefox addon store. The links given in Tiddlywiki documentation is dead. The tiddlyfox repo doesn't offer a xpi to download. Even the build tools required to build the tiddlyfox comes with big red depreciated banner. 

So even if we assume that someone downloads firefox <57 anew, where will he download tiddlyfox from? 

That being the case, I recommend that the following tiddlers be removed from the documentation.

1. [Saving with TiddlyFox](https://tiddlywiki.com/#Saving%20with%20TiddlyFox)
1. [Saving with TiddlyFox on Android](https://tiddlywiki.com/#Saving%20with%20TiddlyFox%20on%20Android)
2. [GettingStarted - Firefox](https://tiddlywiki.com/#GettingStarted%20-%20Firefox)
3. The Tiddlyfox portion of [GettingStarted - Android](https://tiddlywiki.com/#GettingStarted%20-%20Android)

**Comment by pmario** [comment]
See:  Removing archaic tiddly fox references. #6245

**Reply by ibnishak** [reply]
Even then, where can tiddlyfox be found? The links are all dead.

**Reply by pmario** [reply]
I'm with you. I think it should be removed and existing link tiddlers should only show a deprecated info 1 line would be enough
